code/index.html

```diff
@@ -1,22 +1,87 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <title>The News Site</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta name="viewport" content="widht= device-width,initial-scale= 1.0">  
+    <style>@import url('https://fonts.googleapis.com/css2?family=Indie+Flower&display=swap');</style> 
+    <style>@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,300;0,500;0,600;1,500;1,600&display=swap');</style>
   </head>
```

Comment:
when adding custom fonts to your web project, use the link tag instead.
Read more [here](https://developers.google.com/fonts/docs/getting_started)